build/export-openapi.sh

```diff
@@ -73,6 +73,12 @@ mkdir orig
 cp *.json orig
 rm openapi.json
 
+echo "Compiling replaceRef.go..."
```

Comment:
I know you aren't there yet, but once we have a comprehensive go script, we can delete export-openapi.sh